web-admin/src/features/projects/user-management/UsergroupItem.svelte

```diff
@@ -35,7 +36,9 @@
     on:blur={() => (isHovered = false)}
   >
     <UsergroupAvatar {organization} usergroup={group.groupName} />
-    <UsergroupSetRole {organization} {project} {group} />
+    {#if !canManage}
```

Comment:
Is this reversed? `canManage` should be able to set role, right?

Reply:
`canManage` is to trigger the user management dialog when manage text is clicked.

Reply:
the feature is commented out right now